lib/validation.rb

```diff
@@ -30,6 +30,10 @@ def initialize(details)
 
   ALLOWED_PORT_RANGE_PATTERN = '\A(\d+)(?:\.\.(\d+))?\z'
 
+  # - Max length 63
+  # - Alphanumeric, hyphen, underscore, space, parantheses, exclamation, question mark, star
+  ALLOWED_SHORT_TEXT_PATTERN = '\A[a-zA-Z0-9_\-!?\*\(\) ]{1,63}\z'
```

Comment:
dumb question, why are these patterns not Regexp.new / Regexp.compile / %r{} literals that result in a regexp object, rather than strings that presumably have to get compiled over and over?

Reply:
No idea. Probably the first one is added like this and the rest followed the pattern. I added a separate commit to change them to Regexp.new